department-of-veterans-affairs/vets-website
# Mhv 58512 point download all to mhv classic

## Summary
A section for downloading all records with a description and a link to MHV classic has been added to the landing page. 

## Related issue(s)
https://jira.devops.va.gov/browse/MHV-58512
### Remove the "Download all records page" and add link to MHV classic

> Description
> Remove the "Download all records page" and add link to MHV classic
> 
> We are removing the "Download all records page" for phase 0, and instead, linking users back to MHV classic to download their comprehensive medical records reports. In the place of the download section on the landing page that WOULD have linked to the download page (much like the other sections on the landing page that link to the other domains), we are putting a section with a title, a paragraph of text, and a link back to MHV classic. Here is a screenshot of that section and the content. Please refer to this figma page for placement on the landing page: https://www.figma.com/design/mGZRdLypKGaFsHo5xp2kaZ/Medical-Records?node-id=5066-32638&t=n0JZcKxJqPkNvK6c-0 It shoudl be under the settings section and before the "What to know as you try out this tool" section.
> The link should take users to this URL on MHV Classic: https://www.myhealth.va.gov/download-my-data
> 
>  
> 
> User Story: As a Medical Records User, I want to   * , so that I can    
> 
> GIVEN
> 
> WHEN
> 
> THEN
> 
>  
> 
> Feature Flag Y/N ? N
> 
> DataDog/Analytics Y/N ? N
> 
> Manual Testing Y/N ? Y
> 
> Automated Testing Y/N ? Y
> 
> Accessibility Testing Y/N ? Y
> 
> UCD Validation Y/N Y
> 
>  
> 
> Acceptance Criteria:
> 
> AC1   Remove the "Download all records page" and update the UI as shown in FIGMA
> 
> AC2  Link to MHV classic has been added and navigation has been tested with all login options
> 
> AC3
> AC4
> AC5
> 
>  
> 
> Links to UCD:
> 
> Figma Link:  https://www.figma.com/design/mGZRdLypKGaFsHo5xp2kaZ/Medical-Records?node-id=5066-32638&t=n0JZcKxJqPkNvK6c-0
> 
> User Flow:
> 
>  
> 
> Mobile:
> 
> Desktop:
> 
> Other Design Notes:
> 
>  
> 
> Architectural Notes: Always check for the latest designs in Figma

## Testing done
Unit tests were written for the new landing page section and also for download all records helpers to ensure unit test coverage for all metrics stayed above 80%. 

## Screenshots
<img width="858" alt="Screenshot 2024-06-05 at 12 00 28 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/107576133/09d8675b-f50b-44bc-b61b-14b8f09dd95f">
<img width="526" alt="Screenshot 2024-06-05 at 12 00 41 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/107576133/31ae9581-1951-4fb9-b48f-5d70bd27d470">
<img width="1728" alt="Screenshot 2024-06-05 at 11 55 04 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/107576133/9c87018e-71e7-4469-95b8-0f9971733faa">

## What areas of the site does it impact?
MHV medical records landing page

## Acceptance criteria
AC1   Remove the "Download all records page" and update the UI as shown in FIGMA
AC2  Link to MHV classic has been added and navigation has been tested with all login options

### Quality Assurance & Testing


### Error Handling


### Authentication
